src/layouts/uk105.rs

```diff
@@ -53,14 +53,67 @@ impl KeyboardLayout for Uk105Key {
                     DecodedKey::Unicode('4')
                 }
             }
-            KeyCode::HashTilde => {
+            KeyCode::BackSlash => {
```

Comment:
Can you add the docs that you have in the original issue? I pulled up pictures of the UK105 keyboard and was still confused about why this change was necessary. Maybe link back to the original issue as well?